README.md

```diff
@@ -251,6 +251,52 @@ of
 [rqlite](https://rqlite.io/docs/guides/backup/#automatic-backups). Thank
 you to the authors for their great design and documentation.
 
+### Permissions
+
+By default, only the owner / creator of an `ayb` database can access
+it. It's possible to share `ayb` databases in two ways:
+* By setting the public sharing level of the database to give any entity access to the database.
+* By sharing the database with a particular entity.
+
+To set the public sharing level of a database, select one of the following options:
+```
+
+# The default setting: no entity will be able to access the database
+# (unless they specifically get permissions).
+ayb client update_database marcua/test.sqlite --public-sharing-level no-access
```

Comment:
Make these consistent with the other CLI examples (e.g., prefix with $, consider showing output). Not sure about showing output as the commands themselves are clear.